src/main/scala/io/iohk/ethereum/network/p2p/messages/package.scala

```diff
@@ -3,11 +3,29 @@ package io.iohk.ethereum.network.p2p
 import io.iohk.ethereum.network.p2p.Message.Version
 
 package object messages {
-  object Versions {
+  object ProtocolVersions {
     val PV61: Version = 61
     val PV62: Version = 62
     val PV63: Version = 63
+    val PV64: Version = 64
 
     val SubProtocolOffset = 0x10
   }
+
+  object Codes {
+    val StatusCode: Int = ProtocolVersions.SubProtocolOffset + 0x00
+    val NewBlockHashesCode: Int = ProtocolVersions.SubProtocolOffset + 0x01
+    val SignedTransactionsCode: Int = ProtocolVersions.SubProtocolOffset + 0x02
+    val GetBlockHeadersCode: Int = ProtocolVersions.SubProtocolOffset + 0x03
+    val BlockHeadersCode: Int = ProtocolVersions.SubProtocolOffset + 0x04
+    val GetBlockBodiesCode: Int = ProtocolVersions.SubProtocolOffset + 0x05
+    val BlockBodiesCode: Int = ProtocolVersions.SubProtocolOffset + 0x06
+    val NewBlockCode: Int = ProtocolVersions.SubProtocolOffset + 0x07
+    // This message is removed in PV62 and this code is reused in PV65 with different msg type
```

Comment:
Do you see any problem with that? In fact this is what we are doing for `Status` and `NewBlock`. Just because we are calling them the same (names are irrelevant on the protocol level) doesn't change the fact that they are different messages :)

Reply:
I don't know, maybe they know that that message is so old and used only in one protocol version that it was prettier to not block that code forever

Reply:
Nono, I was just complaining about why ETH did this:
>     // This message is removed in PV62 and this code is reused in PV65 with different msg type